src/pyconverter/xml2py/ast_tree.py

```diff
@@ -2009,12 +1948,32 @@ def py_args(self):
         return [arg.lower() for arg in self.args]
 
     @property
-    def py_signature(self):
+    def is_archived(self):
+        """Return whether the command is archived."""
```

Comment:
```suggestion
        """Flag indicating if the command is archived."""
```
Format for a property description.